### PR TITLE
[type:fix] Polish Code.

### PR DESCRIPTION
**Pull request by GOODBOY008**
[type:fix] Polish Code.

### CODE REVIEW

**Comment by dragon-zhang** [comment]
Hi, please check CI

File: shenyu-sync-data-center/shenyu-sync-data-zookeeper/src/main/java/org/apache/shenyu/sync/data/zookeeper/ZookeeperClient.java [hunk 1]
```diff
@@ -226,7 +226,7 @@ public TreeCache getCache(final String path) {
     public TreeCache addCache(final String path, final TreeCacheListener... listeners) {
         TreeCache cache = TreeCache.newBuilder(client, path).build();
         caches.put(path, cache);
-        if (listeners != null && listeners.length > 0) {
+        if (listeners != null) {
```
Comment (moremind) [inline shenyu-sync-data-center/shenyu-sync-data-zookeeper/src/main/java/org/apache/shenyu/sync/data/zookeeper/ZookeeperClient.java:229]: use Objects.isNull

File: shenyu-register-center/shenyu-register-client-server/shenyu-register-client-server-zookeeper/src/main/java/org/apache/shenyu/register/client/server/zookeeper/ZookeeperClient.java [hunk 2]
```diff
@@ -223,7 +223,7 @@ public TreeCache getCache(final String path) {
     public TreeCache addCache(final String path, final TreeCacheListener... listeners) {
         TreeCache cache = TreeCache.newBuilder(client, path).build();
         caches.put(path, cache);
-        if (listeners != null && listeners.length > 0) {
+        if (listeners != null) {
```
Comment (moremind) [inline shenyu-register-center/shenyu-register-client-server/shenyu-register-client-server-zookeeper/src/main/java/org/apache/shenyu/register/client/server/zookeeper/ZookeeperClient.java:226]: use Objects.isNull

File: shenyu-register-center/shenyu-register-client/shenyu-register-client-zookeeper/src/main/java/org/apache/shenyu/register/client/zookeeper/ZookeeperClient.java [hunk 3]
```diff
@@ -223,7 +223,7 @@ public TreeCache getCache(final String path) {
     public TreeCache addCache(final String path, final TreeCacheListener... listeners) {
         TreeCache cache = TreeCache.newBuilder(client, path).build();
         caches.put(path, cache);
-        if (listeners != null && listeners.length > 0) {
+        if (listeners != null) {
```
Comment (moremind) [inline shenyu-register-center/shenyu-register-client/shenyu-register-client-zookeeper/src/main/java/org/apache/shenyu/register/client/zookeeper/ZookeeperClient.java:226]: Objects.isNull

**Review by moremind — CHANGES_REQUESTED** [review]

**Review by moremind — DISMISSED** [review]

**Comment by moremind** [comment]
@GOODBOY008 Hi, pls check ci

**Review by moremind — APPROVED** [review]

**Pull request merged**